testing/upload_validation.ts

```diff
@@ -92,6 +92,15 @@ import * as z from 'zod';
  */
 export const PACKS_VALID_COLUMN_FORMAT_MATCHER_REGEX = /^\/(.*)\/([a-z]+)?$/;
 
+export const Limits = {
+  BuildingBlockCountPerType: 100,
+  BuildingBlockName: 50,
+  BuildingBlockDescription: 100,
+  ColumnMatcherRegex: 100,
```

Comment:
As discussed over chat, these character limits feel a little low. The regex one in particular, given that regular expressions are known to get quite verbose sometimes.

Reply:
Yeah especially if we're allowing 253 characters in the network domain
I think we can set this to 300 or something? Most domains won't be 253 characters long, but just in case it is this gives a bit of leeway